Example/Emission/AppDelegate.m

```diff
@@ -10,7 +10,7 @@
 #import <React/RCTUtils.h>
 #import <TargetConditionals.h>
 
-#define ARTIST @"banksy"
+#define ARTIST @"alex-katz"
```

Comment:
Yeah, I did :D But then I found some QA issues with this artist, so decided to leave it in for now.